### ISSUE TITLE
ViewRender doesn't update on changes to ComponentRigidbody

### ISSUE THREAD

**Issue by JirkaDellOro** [hs-furtwangen/FUDGE#5]
Introduced with commit
"add external resources status handling" on 15/2/2024

Line 287 in ViewComponent was deactivated
```typescript
         this.dispatch(EVENT_EDITOR.UPDATE, { bubbles: true, detail: { node: this.node } }); // TODO: check if this was necessary, EVENT_EDITOR.UPDATE gets broadcasted by project on ƒ.EVENT.GRAPH_MUTATED, so this was causing a double broadcast of EVENT_EDITOR.UPDATE to ALL views on any change to any component
```

The second update actually gets broadcasted only to the graph panel holding the ViewComponent. Maybe there is a way to check if necessary...

**Comment by JirkaDellOro** [comment]
I created a new branch devJirka and do my fixes there, please review

**Comment by plojo** [comment]
We still get multiple `EVENT_EDITOR.UPDATE` broadcasts on changes to `ComponentRigidbody`:

- First, `EVENT_EDITOR.UPDATE` is broadcast by `Project` to all panels. This causes `ViewRender` to redraw, as well as other views like `ViewPreview` to refresh.
- Then, `ViewComponent` dispatches a bubbling `EVENT_EDITOR.UPDATE`. This causes `ViewRender` to redraw again, but this time with the correctly initialized `ComponentRigidbody`. This event now bubbles up to the `Page`, which broadcasts it, skipping `PanelGraph` but not the other panels, so `ViewPreview` refreshes again.
So basically, we end up with all listeners to `EVENT_EDITOR.UPDATE` being called twice in succession.

**Comment by JirkaDellOro** [comment]
Not too beautiful, but low priority, I'd say
